packages/app-root/src/app/users/[login]/stats/page.js

```diff
@@ -0,0 +1,9 @@
+'use client'
```

Comment:
You mentioned not loving this implementation of `'use client'`, but I agree it's definitely fine to do for now! Our frontend stack is dependent on internal and external libraries with client-side React features, so marking an entire page as a client component isn't the end of the world at all. Also, a user's stats page is very dynamic data on top of the fact we don't have server-side authentication to handle display of stats on the server. Just wanted to encourage you not to dwell too much on this `use client`. It's difficult to utilize server component without rebuilding Zooniverse from the ground up 💁‍♀️

Reply:
Thanks that's helpful to hear!